riscv/src/riscv_rvd.rs

```diff
@@ -353,4 +353,157 @@ impl Rvd {
             self.opcodes.insert(115, info);
         }
     }
+
+    // Converta a compressed register index (e.g. rs1') to a full register index (e.g. rs1)
```

Comment:
```suggestion
    // Convert a compressed register index (e.g. rs1') to a full register index (e.g. rs1)
```

Reply:
Solved and pushed, as "Converts...", the original intent.